tests/test_subscriptions.py

```diff
@@ -28,15 +28,21 @@
 logger.setLevel(logging.INFO)
 
 
-class TestSubscriptions(unittest.TestCase, DSSAsserts):
-    @classmethod
-    def setUpClass(cls):
-        cls.es_server = ElasticsearchServer()
-        os.environ['DSS_ES_PORT'] = str(cls.es_server.port)
+class ESInfo:
+    server = None
+
+def setUpModule():
+    ESInfo.server = ElasticsearchServer()
+    os.environ['DSS_ES_PORT'] = str(ESInfo.server.port)
+
+def tearDownModule():
+    ESInfo.server.shutdown()
```

Comment:
i wonder if it makes sense to unset os.environ['DSS_ES_PORT']

Reply:
That sounds like a sensable idea.